src/Umbraco.Web.UI.Client/src/views/propertyeditors/rte/rte.controller.js

```diff
@@ -57,39 +59,32 @@ angular.module("umbraco")
                 var baseLineConfigObj = {
                     maxImageSize: editorConfig.maxImageSize,
                     width: width,
-                    height: height,
-                    init_instance_callback: function(editor){
-                        $scope.isLoading = false;
-                    }
+                    height: height
                 };
-
-                angular.extend(baseLineConfigObj, standardConfig);
-
+                
                 baseLineConfigObj.setup = function (editor) {
-
+                    
                     //set the reference
                     tinyMceEditor = editor;
-
+                    
+                    tinyMceEditor.on('init', function (e) {
```

Comment:
Curious as to why the callback/function approach with the property `init_instance_callback` was not the right approach and to use/listen with the init event instead?

Reply:
The problem with using the callback is that it only allows for one callback. And it was overwriting a callback set by the tinymce.service, so in general to avoid that, we just use events instead.. :-)